README.md

```diff
@@ -18,6 +18,63 @@ To install [the current release](https://pypi.org/project/nvflare), you can simp
 pip install nvflare
 ```
 
+## Quick Start
+
+ * [Getting Started](https://nvflare.readthedocs.io/en/latest/getting_started.html)
```

Comment:
Do we want always put the latest here?
If so then this is fine.

If we want to say most recent releases maybe need to do main, but that could be updated later